scripts/upgrade-test-def-id-type.ts

```diff
@@ -0,0 +1,41 @@
+import { DeployHelper } from "../helpers/DeployHelper";
+import { ethers } from "hardhat";
+import { StateContractMigrationHelper } from "../helpers/StateContractMigrationHelper";
+import { NetworkIdTypes } from "../helpers/NetworkIdTypes";
+/*
+1. deploy stateV2 to mumbai from feature/state-v3 branch
+2. run transit-state script
+3. cp .openzeppelin/* ../../contracts/.openzeppelin/
+4. update addreess and block number in data
+5. run this script
+*/
+
+async function main() {
+
+    const signers = await ethers.getSigners();
+    const stateDeployHelper = await DeployHelper.initialize(null, true);
+    const stateContractMigrationHelper = new StateContractMigrationHelper(stateDeployHelper, signers[0]);
+
+    const oldContractABI = require("./StateV2_deployed_abi.json");  // abi of contract that will be upgraded
+    const stateContractAddress = "0x316345cC62A81bfD5dAe4f37FFBD07FD95b4f51c";  // address of contract that will be upgraded
```

Comment:
Better to leave it empty to make people explicitly insert contracts here at before the run.